README.Rmd

```diff
@@ -33,46 +33,47 @@ remotes::install_github("jhudsl/mario")
 ## Example
 
 Before you can run `mario`, you will need two things:
-1) An API Key and
-2) Your Google slide ID that you'd like to translate into a video.
+
+1) An API Key
+2) Your Google slide ID that you'd like to translate into a video
```

Comment:
Maybe link down to below sections here? Or better yet, an auto-generated Table of Contents